install.py

```diff
@@ -445,6 +445,8 @@ def validate_path(path):
 -DLegion_EMBED_GASNet_CONFIGURE_ARGS="--with-ibv-max-hcas=8"
 """.splitlines()
 
+    if cuda:
+        cmake_flags += [f"-DLegion_CUDA_ARCH={arch}"]
```

Comment:
I probably missed some discussion on this. Why the switch from `CMAKE_CUDA_ARCHITECTURES` to `Legion_CUDA_ARCH`?

Reply:
See above.